### PR TITLE
Add TSC <> CommComm Docs for Working Groups

### PR DESCRIPTION
**Pull request by bnb**
A first pass at documentation for transferring between the TSC and the Community Committee (and the reverse process, as well). Some of this is conjecture, but I wanted to take a pass at building out some defined process around the... process of moving from one committee to another. 😄

This PR also includes two issue templates to help with creating issues in the specific WG. Creating templates for the issues to be submitted to the CommComm/TSC is definitely something that we could do as well.

### CODE REVIEW

File: documents/moving-across-committees.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Transfering a Working Group from the Node.js TSC to the Node.js Community Committee
+
+As interested, Working Groups that are focused on serving, growing, and building the community surrounding Node.js may decide that it would like to become a part of the Node.js Community Committee. Similarly, if a Working Group under the Node.js Community Committee is focused on more technical tasks and may want to transfer over to the TSC.
+
+This document is a guide on how to begin the transfer process from one committee to another.
+
+## Moving from the TSC to the Community Committee
+* **Step 1**: Create an issue in [Community Committee](https://github.com/nodejs/community-committee/issues) repository asking if the Community Committee would be willing to take the Working Group in question on as a Community Committee Working Group.
```
Comment (pup) [inline documents/moving-across-committees.md]: `in the`

File: documents/moving-across-committees.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  * **Step 1.1**: If there are no objections, continue on to step two.
```
Comment (pup) [inline documents/moving-across-committees.md]: is there a plan if the commcomm denies the transferral request?

File: documents/moving-across-committees.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+* **Step 2**: Create an issue in the repository for the Working Group in question, proposing the move change. Given the governance model, this will likely be a vote and will need majority approval of the Working Group's members.
```
Comment (pup) [inline documents/moving-across-committees.md]: `of the Working Group's members`? i think `from` sounds better in this context

File: documents/moving-across-committees.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+* **Step 3**: Once approved, you will need to create issues to be removed from the TSC and added to the Community Committee, and an issue tracking the change in the relevant WG repo. 
+  * **Step 3.1**: Create an issue to track the changes in the Working Group's repository. There is a template to help you with this that can be found [here](./templates/tsc_to_commcomm.md).
+  * **Step 3.2**: Create an issue requesting to be chartered in the [Community Committee repo](https://github.com/nodejs/community-committee/issues). Be sure to link to the issue in the WG.
+  * **Step 3.3**: Create an issue requesting to be dechartered in the [TSC](https://github.com/nodejs/tsc/issues). Be sure to link to the issue in the WG.
+* **Step 4**: Once dechartered by the TSC and chartered by the Community Committee, the process is complete.
+
+## Moving from the Community Committee to the TSC
+* **Step 1**: Create an issue in [TSC](https://github.com/nodejs/tsc/issues) repository asking if the Community Committee would be willing to take the Working Group in question on as a TSC Working Group.
+  * **Step 1.1**: If there are no objections, continue on to step two.
```
Comment (pup) [inline documents/moving-across-committees.md]: same as above?

File: documents/moving-across-committees.md (hunk 2 above)
Comment (nebrius) [inline documents/moving-across-committees.md]: the group requesting can either address the feedback and try again, or stay where they are.

File: documents/moving-across-committees.md (hunk 3 above)
Comment (nebrius) [inline documents/moving-across-committees.md]: I think this should be step one, not step two. Asking to move a WG before asking if that's something the WG wants to do has the potential to cause a lot of conflict and confusion.

File: documents/moving-across-committees.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+This document is a guide on how to begin the transfer process from one committee to another.
+
+## Moving from the TSC to the Community Committee
+* **Step 1**: Create an issue in [Community Committee](https://github.com/nodejs/community-committee/issues) repository asking if the Community Committee would be willing to take the Working Group in question on as a Community Committee Working Group.
+  * **Step 1.1**: If there are no objections, continue on to step two.
+* **Step 2**: Create an issue in the repository for the Working Group in question, proposing the move change. Given the governance model, this will likely be a vote and will need majority approval of the Working Group's members.
+* **Step 3**: Once approved, you will need to create issues to be removed from the TSC and added to the Community Committee, and an issue tracking the change in the relevant WG repo.
```
Comment (nebrius) [inline documents/moving-across-committees.md]: replace `issue` with `pull request`, and there's no need for a separate issue in the WG repo. We should use the original issue creating in what I suggest be step 1. Otherwise, we're creating notification clutter.

File: documents/moving-across-committees.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+
+This document is a guide on how to begin the transfer process from one committee to another.
+
+## Moving from the TSC to the Community Committee
+* **Step 1**: Create an issue in [Community Committee](https://github.com/nodejs/community-committee/issues) repository asking if the Community Committee would be willing to take the Working Group in question on as a Community Committee Working Group.
+  * **Step 1.1**: If there are no objections, continue on to step two.
+* **Step 2**: Create an issue in the repository for the Working Group in question, proposing the move change. Given the governance model, this will likely be a vote and will need majority approval of the Working Group's members.
+* **Step 3**: Once approved, you will need to create issues to be removed from the TSC and added to the Community Committee, and an issue tracking the change in the relevant WG repo. 
+  * **Step 3.1**: Create an issue to track the changes in the Working Group's repository. There is a template to help you with this that can be found [here](./templates/tsc_to_commcomm.md).
```
Comment (nebrius) [inline documents/moving-across-committees.md]: I would drop this step entirely. We already have one issue to track things.

File: documents/moving-across-committees.md [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+  * **Step 3.2**: Create an issue requesting to be chartered in the [Community Committee repo](https://github.com/nodejs/community-committee/issues). Be sure to link to the issue in the WG.
```
Comment (nebrius) [inline documents/moving-across-committees.md]: This step was already done in what I'm suggesting be step 2 (currently step 1), so we can eliminate this step.

File: documents/moving-across-committees.md [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+  * **Step 3.3**: Create an issue requesting to be dechartered in the [TSC](https://github.com/nodejs/tsc/issues). Be sure to link to the issue in the WG.
```
Comment (nebrius) [inline documents/moving-across-committees.md]: replace `issue` with `pull request`.

File: documents/moving-across-committees.md [hunk 9]
```diff
@@ -0,0 +1,25 @@
...
+* **Step 4**: Once dechartered by the TSC and chartered by the Community Committee, the process is complete.
+
+## Moving from the Community Committee to the TSC
```
Comment (nebrius) [inline documents/moving-across-committees.md]: I would remove this entire section and adjust the one above to read "moving from one top level committee to another"

File: documents/templates/commcomm_to_tsc.md [hunk 10]
```diff
@@ -0,0 +1,7 @@
+```
```
Comment (nebrius) [inline documents/templates/commcomm_to_tsc.md]: TBH we don't need, and I don't think we should have, templates for this. Each move is going to be pretty one off, and the _actual_ steps involved will vary from transfer to transfer. Using a template encourages fitting a square peg in a round hole.

File: documents/moving-across-committees.md (hunk 2 above)
Comment (bnb) [inline documents/moving-across-committees.md]: Would you like me to note as such in this section?

File: documents/templates/commcomm_to_tsc.md (hunk 10 above)
Comment (bnb) [inline documents/templates/commcomm_to_tsc.md]: >Using a template encourages fitting a square peg in a round hole.

Definitely agree on there being an issue with that - my main concern is that creating an issue in the CC and the TSC repos could honestly be a barrier to those interested in this process, but I may be mixing optimizing the experience of someone _new_ to the project versus someone who is already active and is aware of these processes.

Do you think this could be a possible barrier? If not, I'm +1 to completely removing these templates.

File: documents/moving-across-committees.md (hunk 9 above)
Comment (bnb) [inline documents/moving-across-committees.md]: My concern with this is that the Community Committee then would become the de facto place for documentation on this to live, without input from other top-level committees. As I see it, the way that could taken is that the Community Committee is asserting that it is an authority on _all_ top-level committees, and has sole control over the guiding documentation around this. 

I am 100% for abstracting this, but I do want to make sure that this isn't seen as an assertion of one group having any kind of power over another, without the other group's consent.

File: documents/moving-across-committees.md (hunk 2 above)
Comment (pup) [inline documents/moving-across-committees.md]: preferably!

File: documents/templates/commcomm_to_tsc.md (hunk 10 above)
Comment (nebrius) [inline documents/templates/commcomm_to_tsc.md]: I don't think it's a realistic barrier, no, because I cannot envision a scenario where someone new to the project would be using these steps. I would even go so far as to say that someone new to this project _shouldn't_ follow these steps. The person following these steps should be the leader of the WG that wants to move, and becoming a leader of a WG isn't going to be someone new to the project (or at least new enough to not be able to file an issue).

Also, this process will be used _very_ infrequently, and whenever we're discussing changes like this it's big. This means there will be multiple TSC and CommComm members available to help with the request.

File: documents/moving-across-committees.md [hunk 11]
```diff
@@ -0,0 +1,33 @@
...
+
+### Moving from the TSC to the Community Committee
+* **Step 1**: Create an issue in the repository for the Working Group in question, proposing the move. Given the governance model, this will likely be a vote and will need majority approval from the Working Group's members.
+* **Step 2**: Create an issue in the [Community Committee](https://github.com/nodejs/community-committee/issues) repository asking if the Community Committee would be willing to take the Working Group in question on as a Community Committee Working Group.
+  * **Step 2.1**: If there _are no objections_, continue on to step two.
+  * **Step 2.2**: If there _are objections_ by the committee, the group requesting the transfer can either address feedback and try again or remain under their current committee.
+* **Step 3**: Once approved by both the WG and the Community Committee, you will need to create two pull requests: one pull request to be dechartered by the TSC and one pull request to be chartered by the Community Committee.
+* **Step 4**: Once dechartered by the TSC and chartered by the Community Committee, the process is complete.
+
+### Moving from the Community Committee to the TSC
```
Comment (nebrius) [inline documents/moving-across-committees.md]: I still really think we should rework this to be a single list that is "from one committee to another" insetad of separating them out. Otherwise you'll need to duplicate this four more times because we're currently missing how to move WGs from the CTC, and that's a maintenance headache to maintain.

File: documents/moving-across-committees.md [hunk 12]
```diff
@@ -0,0 +1,33 @@
...
+- [] Have the currrent committee vote to decharter
+- [] Have the new committee vote (and approve) a new charter
+- [] Mark that the Working Group has been dechartered in the Working Groups documentation for the current committee
+- [] Submit a PR to the new committee adding the Working Group to the Working Groups documentation for the new committee
+
+## In depth: A guide to moving from one commmitte
+While the process of moving from one top level committee to another is likely going to be one of the smoother parts of this process, we've documented it in case you want some help with going through the process of this transition.
+
+### Moving from the TSC to the Community Committee
+* **Step 1**: Create an issue in the repository for the Working Group in question, proposing the move. Given the governance model, this will likely be a vote and will need majority approval from the Working Group's members.
```
Comment (nebrius) [inline documents/moving-across-committees.md]: Funny thing is, it's actually pretty unlikely to be a vote. Charter changes are not called out specifically as requiring a vote, meaning it falls to the consensus-seeking model. Most issues resolved using the consensus seeking model don't go to a vote. That assumes that the WG uses the generic consensus seeking model though and that they copied that part of the charter from the charter template in the TSC repo, but it could be something else. We should leave that part out, as it's up to the WG to decide how they decide and vote on things.

I would reword this to say "Create an issue in the repository for the Working Group in question and gain approval from the working group for the move."

File: documents/moving-across-committees.md [hunk 13]
```diff
@@ -0,0 +1,33 @@
...
+* **Step 2**: Create an issue in the [Community Committee](https://github.com/nodejs/community-committee/issues) repository asking if the Community Committee would be willing to take the Working Group in question on as a Community Committee Working Group.
+  * **Step 2.1**: If there _are no objections_, continue on to step two.
```
Comment (nebrius) [inline documents/moving-across-committees.md]: I feel like this is implicit, but if you want to explicitly call out the "if there are/are not objects, do x," then we should be consistent and add it to step 1 too.

File: documents/moving-across-committees.md [hunk 14]
```diff
@@ -0,0 +1,33 @@
...
+  * **Step 2.2**: If there _are objections_ by the committee, the group requesting the transfer can either address feedback and try again or remain under their current committee.
+* **Step 3**: Once approved by both the WG and the Community Committee, you will need to create two pull requests: one pull request to be dechartered by the TSC and one pull request to be chartered by the Community Committee.
+* **Step 4**: Once dechartered by the TSC and chartered by the Community Committee, the process is complete.
```
Comment (nebrius) [inline documents/moving-across-committees.md]: The steps aren't complete at this point, there's the checklist above that also needs to be completed. I would integrate the checklist into this part of the list.

File: documents/moving-across-committees.md (hunk 13 above)
Comment (bnb) [inline documents/moving-across-committees.md]: I think this got mangled somehow - notice it says "step two". Not sure what happened there - maybe updated locally after this. Update incoming.

File: documents/moving-across-committees.md (hunk 14 above)
Comment (bnb) [inline documents/moving-across-committees.md]: Working on this, want to make sure I've got a complete understanding: Update the checklist to fit in here - do you think I should remove or keep it above?

File: documents/moving-across-committees.md [hunk 15]
```diff
@@ -1,6 +1,5 @@
-# Transfering a Working Group from the Node.js TSC to the Node.js Community Committee
-
-As interested, Working Groups that are focused on serving, growing, and building the community surrounding Node.js may decide that it would like to become a part of the Node.js Community Committee. Similarly, if a Working Group under the Node.js Community Committee is focused on more technical tasks and may want to transfer over to the TSC.
+# Transfering a Working Group from One Node.js Commmittee to Another
```
Comment (nebrius) [inline documents/moving-across-committees.md]: Spelling nit: Remove extra "m" in Commmittee

File: documents/moving-across-committees.md [hunk 16]
```diff
@@ -16,18 +15,16 @@ Here's a checklist for you to get started with transferring committees. This lis
...
+### Moving from one Committee to another Committee
 
-### Moving from the Community Committee to the TSC
-* **Step 1**: Create an issue in the repository for the Working Group in question, proposing the move. Given the governance model, this will likely be a vote and will need majority approval from the Working Group's members.
-* **Step 2**: Create an issue in the [TSC](https://github.com/nodejs/TSC/issues) repository asking if the TSC would be willing to take the Working Group in question on as a TSC Working Group.
-  * **Step 2.1**: If there _are no objections_, continue on to step two.
+* **Step 1**: Create an issue in the repository for the Working Group in question and gain approval from the Working Group for the move.
+  * **Step 1.1**:If there _are no objections_, continue on to Step 2.
+* **Step 2**: Create an issue in the repository of the new Committee asking if it would be willing to take the Working Group in question on as a Working Group of that Committee.
+  * **Step 2.1**: If there _are no objections_, continue on to step 2.2.
```
Comment (nebrius) [inline documents/moving-across-committees.md]: Should this read "continue on to step 3"?

File: documents/moving-across-committees.md [hunk 17]
```diff
@@ -16,18 +15,16 @@ Here's a checklist for you to get started with transferring committees. This lis
...
   * **Step 2.2**: If there _are objections_ by the committee, the group requesting the transfer can either address feedback and try again or remain under their current committee.
-* **Step 3**: Once approved by both the WG and the TSC, you will need to create two pull requests: one pull request to be dechartered by the Community Committee and one pull request to be chartered by the TSC.
-* **Step 4**: Once dechartered by the Community Committee and chartered by the TSC, the process is complete.
+* **Step 3**: Once approved by both the WG and the new Committee, you will need to create two pull requests: one pull request to be dechartered by the previous Committee and one pull request to be chartered by the new Committee.
+* **Step 4**: Once dechartered by the previous Committee and chartered by the new Committee, you can use the following checklist 
+  * [] Have the currrent committee vote to decharter
+  * [] Have the new committee vote (and approve) a new charter
```
Comment (nebrius) [inline documents/moving-across-committees.md]: The "(and approve)" bit is superfluous IMO. A vote intrinsically means approving or dissaproving of a thing following an ordered process. I would remove it.

**Comment by bnb** [comment]
@nebrius Updated 😄

**Comment by bnb** [comment]
@nebrius Can this be merged? 😊

**Comment by nebrius** [comment]
Yes, thanks for the reminder @bnb!
